core/shared/src/main/scala/com/monovore/decline/Help.scala

```diff
@@ -49,11 +49,17 @@ object Help {
       else ("Options and flags:" :: optionsDetail).mkString("\n") :: Nil
     }
 
+    var envVarHelp = {
+      val envVarNames = environmentVars(parser.options)
+      if (envVarNames.isEmpty) Nil
+      else ("Environment Variables:" :: envVarNames.map("    " ++ _)).mkString("\n") :: Nil
+    }
+
     Help(
       errors = Nil,
       prefix = NonEmptyList(parser.name, Nil),
       usage = Usage.fromOpts(parser.options).flatMap { _.show },
-      body = (parser.header :: optionsHelp) ::: commandHelp
+      body = ((parser.header :: optionsHelp) ::: envVarHelp) ::: commandHelp
```

Comment:
I don’t think you need the extra parens here do you? 

Also just one set:
`parser.header :: (optionsHelp ::: envVarHelp ::: commandHelp)` is the most efficient since lists are build from the right to the left.